test/datasets/test_metrla.py

```diff
@@ -0,0 +1,38 @@
+import shutil
+
+from torch_geometric.datasets import MetrLa
+
+
+def test_metrla() -> None:
+    root = '/tmp/metr_la'
+    n_previous_steps = 6
+    n_future_steps = 6
+
+    print("This may take a couple of minutes...")
```

Comment:
How long does it take to download the dataset? We generally do not test downloading and processing datasets as part of our CI. I'm fine with dropping this test completely.

Reply:
Downloading takes like 10 sec, processing takes a good 2-3 minutes. I can try to limit it to generate a single example and do the last 5 asserts on it, otherwise will just drop it.